removeDuplicates.js

```diff
@@ -0,0 +1 @@
+const removeDuplicates = (array) => Array.from(new Set(array));
```

Comment:
Remove brackets. When arrow functions has only one parameter it is good practice to remove brackets  it improves codes readability.